tensorboard/components/tf_markdown_view/tf-markdown-view.html

```diff
@@ -0,0 +1,82 @@
+<!--
+@license
+Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<link rel="import" href="../polymer/polymer.html">
+
+<!--
+  tf-markdown-view renders raw HTML that has been converted from
+  Markdown by some other agent. The HTML must be sanitized, and must be
+  safe to inject directly into the DOM.
+
+  TensorBoard plugins can use the `markdown_to_safe_html` function from
+  the `tensorboard.plugin_util` module on the backend to generate HTML
+  suitable for use with this component.
+-->
+<dom-module id="tf-markdown-view">
+  <template>
+    <div id="markdown" inner-h-t-m-l="[[html]]"></div>
+    <style>
+      /*
+       * Reduce topmost and bottommost margins from 16px to 0.3em (renders
```

Comment:
> 0.3m actually varies based on the font

Of course; that's the point of using `em`s. :-) It renders at exactly 4.8 pixels with our current configuration—which should be relatively stable—so I wrote "about 4.8px" so that people reading the code who aren't intuitively familiar with `em` can estimate the size.